### ISSUE TITLE
Installing on RT1900ac requires a USB disk to be plugged in

### ISSUE THREAD

**Issue by tzhao11** [publicarray/spksrc#4]
For now, installing the package on RT1900ac requires a USB disk to be plugged in. Is it possible to install the package to internal NAND?

**Comment by publicarray** [comment]
Yes unfortunately this is required at the moment, As far as I'm aware Synology Apps are also installed on the USB/ SD card.

**Comment by tzhao11** [comment]
From my testing DNS Server, RADIUS Server and VPN Plus Server can be installed without USB/SD. All other Synology Apps requires a USB disk.
